src/serializer/types.js

```diff
@@ -41,9 +50,13 @@ export type ResidualFunctionBinding = {
   modified: boolean,
   // void means a global binding
   declarativeEnvironmentRecord: null | DeclarativeEnvironmentRecord,
+  // an additional function may overwrite the value of this binding
+  additionalValue?: null | Value,
```

Comment:
This seems unreferenced, and just by the name of it it's not clear to me what it does.

Reply:
This is to put the overwritten value when an additional function overwrites a binding.